Model/User.hs

```diff
@@ -76,14 +75,11 @@ credsToUser eplanId Creds{..} = User
     eprofile = extraToProfile credsPlugin credsExtra
 
 extraToProfile :: Text -> [(Text, Text)] -> Either Text Profile
-extraToProfile "dummy" _ = Right dummyProfile
+extraToProfile "dummy" extra = githubProfile extra
```

Comment:
Dummy logins in development would have an empty mapping, but we don't use these in development.

Dummy logins in test fall into two scenarios:

- Most tests authenticate an existing user (via `authenticateAs`), so the failure to build the profile due to the empty mapping is ignored
- The tests of `authenticateUser` themselves, pass the `Creds` explicitly, so I just provide an appropriate (GitHub-like) mapping